README.rst

```diff
@@ -25,7 +25,7 @@ Features
 - A simple to use API for scheduling jobs.
 - Very lightweight and no external dependencies.
 - Excellent test coverage.
-- Tested on Python 2.7, 3.5, and 3.6
+- Tested on Python 2.7, 3.5, 3.6 and 3.7
```

Comment:
(nitpicky formatting comment): This removes the Oxford comma before the "and" :P